pom.xml

```diff
@@ -3387,6 +3387,8 @@
                   <arg>-Wconf:msg=Auto-application to \`\(\)\` is deprecated:s</arg>
                   <arg>-Wconf:msg=method with a single empty parameter list overrides method without any parameter list:s</arg>
                   <arg>-Wconf:msg=method without a parameter list overrides a method with a single empty one:s</arg>
+                  <!-- SPARK-35496 Suppress `The outer reference in this type test cannot be checked at run time` -->
```

Comment:
Just to try not to modify SQLQueryTestSuite and ThriftServerQueryTestSuite, it may be useful to upgrade Scala 2.13.7 later

Reply:
should be `<arg>-Wconf:cat=unchecked&amp;msg=outer reference:s</arg>`